examples/nirfmxbluetooth/txp-basic.py

```diff
@@ -297,22 +298,24 @@ def raise_if_error(response):
         f"Peak to Average Power Ratio Maximum (dB)        : {peak_to_average_power_ratio_maximum}"
     )
 
-    # Uncomment this block if these values are generated by your measurement:
-    # print(f"EDR GFSK Average Power Mean (dBm)               : {edrgfsk_average_power_mean}")
-    # print(f"EDR DPSK Average Power Mean (dBm)               : {edrdpsk_average_power_mean}")
-    # print(
-    #     f"EDR DPSK GFSK Average Power Ratio Mean (dB)     : {edR__dpsK__gfsk_average_power_ratio_mean}"
-    # )
+    if basic_rate_packets == False:
```

Comment:
It's generally preferred to use "is" instead of "==" when comparing with `True` or `False`. Unless you specifically want to allow falsey values like `0` or `[]`.

Reply:
Or you can bypass the argument altogether by using `if not basic_rate_packets:`

Reply:
I've implemented this suggestion.